chirp/drivers/mml_jc8810.py

```diff
@@ -934,7 +935,16 @@ def apply_skeytop_listvalue(setting, obj):
                 val = SKEYTOP_VALUES[index]
                 obj.set_value(val)
 
-            if self.MODEL in ["RT-470L"]:
+            if self.MODEL in ["RT-470"]:
+                # ==========
+                # Notice to developers:
+                # The RT-470 v1.22 firmware added 'hidden' support for the
+                # Top Key (Short Press) feature. RT-470 radios with a firmware
+                # version prior to v1.22 will not honor the Top Key (Short
+                # Press) setting in CHIRP.
+                # ==========
```

Comment:
This is unfortunate for sure, but I prefer this. Is the firmware update-able by the user or just newer radios shipping with newer stuff?

What about adding a info or experimental prompt for the user explaining that the driver expects the newer firmware and some options will not be honored on older ones?